spec/features/roles_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+feature Role do
+  let!(:conference) { create(:conference) }
```

Comment:
Btw: does this really need to be a feature test? From my POV a feature test is a little bit overhead and a simple model test should also do the trick ...